script.module.resolveurl/lib/resolveurl/plugins/streamlare.py

```diff
@@ -24,8 +24,8 @@
 
 class StreamLareResolver(ResolveUrl):
     name = "StreamLare"
-    domains = ["streamlare.com"]
-    pattern = r'(?://|\.)(streamlare\.com)/(?:e|v)/([0-9A-Za-z]+)'
+    domains = ["streamlare.com", "slmaxed.com"] 
+    pattern = r'(?://|\.)(streamlare\slmaxed\.com)/(?:e|v)/([0-9A-Za-z]+)'
```

Comment:
Pattern is not right, did you test if it works?
It should be like this for it to work
`pattern = r'(?://|\.)((?:streamlare|slmaxed)\.com)/(?:e|v)/([0-9A-Za-z]+)'`

Reply:
I test it and it not work unfortunstely

Reply:
Can you give me a sample embed in slmaxed.com

Reply:
Yes can't think of a way to make it work as they have enabled cloudflare browser fingerprinting
`URLError Error - From: StreamLare Link: https://slmaxed.com/e/XXXXXXXX: <urlopen error [WinError 10054] An existing connection was forcibly closed by the remote host>`